src/topp/InternalCalibration.cpp

```diff
@@ -169,6 +169,7 @@ class TOPPInternalCalibration :
     setValidFormats_("in", ListUtils::create<String>("mzML"));
     registerOutputFile_("out", "<file>", "", "Output file ");
     setValidFormats_("out", ListUtils::create<String>("mzML"));
+    registerInputFile_("executable", "<file>", "Rscript", "Path to the Rscript executable (default: 'Rscript').", false);
```

Comment:
I'd (strongly) vote for `rscript_executable` here since its self explanatory and `executable` alone does not say what kind of executable is meant here (reading the docs is required).